src/gordon_gcp/plugins/enricher.py

```diff
@@ -76,7 +77,7 @@ def _validate_config(self):
             errors.append(msg)
 
         if not self.config.get('dns_zone', '').endswith('.'):
-            msg = 'A dns zone must be a FQDN and end with the root zone (".").'
+            msg = 'A dns zone must be an FQDN and end with the root zone (".").'
             errors.append(msg)
```

Comment:
Actually, it's unrelated to typos, but I don't see the point in fixing typos for something that should change. If you wanted to make a new commit to just drop this whole if-block, and maybe reword the msg in the one above, that would be better, so that it would match the format expected by the janitor https://github.com/spotify/gordon-janitor-gcp/blob/develop/src/gordon_janitor_gcp/plugins/authority.py#L92

Reply:
If I understand you correctly, the janitor one seems like it's missing the FQDN test.  Should we change that side?

Reply:
It was intentionally removed because none of the DNS provider APIs that I checked (Google, DYN, NS1, Route53)  expect a zone identifier in that format

Reply:
I retract the above comment.  I now (again) think it might be a good idea to use the absolute zone name, regardless of what the API's expect, since I think it makes more sense to us logically while reasoning about the system.  I will update `gordon-janitor-gcp`